bot/code_review_bot/__init__.py

```diff
@@ -100,6 +107,45 @@ def is_publishable(self):
 
         raise Exception("Unsupported publication mode {}".format(settings.publication))
 
+    def build_hash(self):
+        """
+        Build a uniquely identifying hash for that issue
+        """
+        assert self.revision is not None, "Missing revision"
+
+        try:
+            # Load all the lines affected by the issue
+            file_content = self.revision.load_file(self.path)
+        except requests.exceptions.HTTPError as e:
+            if e.response.status_code == 404:
+                logger.warning(
+                    "Failed to download a file with an issue", path=self.path
```

Comment:
This should never happen, right? Unless the analyzer is mentioning some build file. Maybe make it an error instead of a warning.

Reply:
So it happens with clang-tidy currently quite a lot (Andi made a patch to fix that in https://phabricator.services.mozilla.com/D48702